assets/js/services/index.tsx

```diff
@@ -1,25 +1,42 @@
+import * as JwtApi from "./jwtApi";
+
 const axios = require("axios").default;
 
-axios.interceptors.request.use(
+const redirect = (suffix: string) => {
+  window.location.href = `${window.location.origin}${suffix}`;
+};
+
+const axiosWithInterceptor = axios.create();
+
+axiosWithInterceptor.interceptors.request.use(
   (config) => {
-    const bearer = `Bearer ${localStorage.getItem("jwt")}`;
-    config.headers.Authorization = bearer;
-    return config;
+    return JwtApi.addJwtToHeader(config);
   },
   (error) => {
     Promise.reject(error);
   }
 );
 
-axios.interceptors.response.use(undefined, (error) => {
-  if ([401, 403].includes(error.response.status)) {
-    localStorage.removeItem("jwt");
+axiosWithInterceptor.interceptors.response.use(
+  (response) => response,
+  async (error) => {
+    const status = error.response ? error.response.status : null;
+    if (status === 401) {
```

Comment:
Couldn't we remove one nesting level by returning `Promise.reject(error)` earlier if the status is different than 401?

Reply:
Sure do ⭐️